zuul.d/project-templates.yaml

```diff
@@ -99,6 +99,22 @@
       queue: integrated
       jobs: *ansible-collections-arista-eos-jobs
 
+- project-template:
+    name: ansible-collections-ansible-utils-units
+    check:
+      jobs: &ansile-collections-ansible-utils-units-jobs
+        - build-ansible-collection
```

Comment:
this is the wrong location, it should be in ansible-collections-ansible-utils project-template

Reply:
but we can move it later